config/rbac/cluster_role.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pulp-manager-cluster-role
+rules:
+  - apiGroups:
+      - config.openshift.io
+    resourceNames:
+      - cluster
+    resources:
+      - ingresses
+    verbs:
+      - get
```

Comment:
Adding this `clusterrole` to be possible to get the default ingress domain in case there is no `route_host` specified.